subprojects/packagefiles/libsass/meson.build

```diff
@@ -0,0 +1,23 @@
+project('libsass', 'c', 'cpp',
+        version: '3.6.4',
+        meson_version : '>= 0.48.0',
+        default_options: [
+            'c_std=c99',
+            'cpp_std=c++11',
+            'buildtype=debugoptimized',
+        ])
+
+add_project_arguments(
+  '-DLIBSASS_VERSION="@0@"'.format(meson.project_version()),
+  language: ['cpp'])
+
+cpp = meson.get_compiler('cpp')
+add_project_arguments(
+  cpp.get_supported_arguments('-Wno-non-virtual-dtor'),
+  language: ['cpp'])
```

Comment:
Meson 1.0.0 removed the `-Wnon-virtual-dtor` that this switches off, so you can probably just drop it entirely. See https://github.com/mesonbuild/meson/pull/10339